src/Glpi/Config.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
I had choose another name to prevent name "conflict" with old legacy object. This is quite hard to be sure we're usin `\Config` or `Config`. Found that confusing.

Reply:
Yep :)